modules/compose/compose_api_test.go

```diff
@@ -365,6 +367,29 @@ func TestDockerComposeAPIWithVolume(t *testing.T) {
 	assert.NoError(t, err, "compose.Up()")
 }
 
+func TestDockerComposeAPIVolumesDeletedOnDown(t *testing.T) {
+	identifier := uuid.New().String()
+	stackFiles := WithStackFiles("./testresources/docker-compose-volume.yml")
+	compose, err := NewDockerComposeWith(stackFiles, StackIdentifier(identifier))
+	assert.NoError(t, err, "NewDockerCompose()")
+
+	ctx, cancel := context.WithCancel(context.Background())
+	t.Cleanup(cancel)
+
+	err = compose.Up(ctx, Wait(true))
+	assert.NoError(t, err, "compose.Up()")
+
+	err = compose.Down(context.Background(), RemoveOrphans(true), RemoveVolumes(true), RemoveImagesLocal)
+	assert.NoError(t, err, "compose.Down()")
+
+	volumeListFilters := filters.NewArgs()
+	volumeListFilters.Add("name", fmt.Sprintf("%s_mydata", identifier))
```

Comment:
NIT: Could you add a comment mentioning that this volume comes from the compose referenced above? 🙏